snpeff/4.1k/Dockerfile

```diff
@@ -0,0 +1,47 @@
+#################################################################
+# Dockerfile
+#
+# Version:          1
+# Software:         snpeff
+# Software Version: 4.1k
+# Description:      Genetic variant annotation and effect prediction toolbox
+# Website:          http://sourceforge.net/projects/snpeff/|http://snpeff.sourceforge.net/
+# Tags:             Genomics
+# Provides:         snpeff 4.1k
+# Base Image:       biodckr/biodocker
+# Build Cmd:        docker build --rm -t biodckrdev/snpeff 4.1k/.
+# Pull Cmd:         docker pull biodckrdev/snpeff
+# Run Cmd:          docker run --rm -it biodckrdev/snpeff <options> <files>
+#################################################################
+
+# Set the base image to Ubuntu
+FROM biodckr/biodocker
```

Comment:
same remarks, should use biocontainers base image and use LABEL for metadata (+ license info)